build.assets/tooling/cmd/notarize-apple-binaries/loggerConfig.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+type LoggerConfig struct {
+	logLevel uint
+	logJSON  bool
+}
+
+func NewLoggerConfig() *LoggerConfig {
+	lc := &LoggerConfig{}
+	flag.UintVar(&lc.logLevel, "log-level", uint(logrus.InfoLevel), "Log level from 0 to 6, 6 being the most verbose")
```

Comment:
I'd move this to `args.go` to have all the flag definitions in the same place. And maybe use `kingpin` to try and take care of validation for you.

Reply:
Moved to args. I'm holding off on using `kingpin` pending the arg validation conversation above.

Reply:
Updated to use kingpin